NSE/Sources/NotificationServiceExtension.swift

```diff
@@ -165,3 +149,18 @@ class NotificationServiceExtension: UNNotificationServiceExtension {
         MXLog.info("\(tag) deinit")
     }
 }
+
+private extension Client {
+    func getNotificationItemProxy(roomID: String, eventID: String) async throws -> NotificationItemProxyProtocol {
```

Comment:
As this is async, no need for the `get`.
```suggestion
    func notificationItemProxy(roomID: String, eventID: String) async throws -> NotificationItemProxyProtocol {
```